titan/assetlist.json

```diff
@@ -238,6 +238,48 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
         }
       ]
+    },
+    {
+      "description": "USDC stablecoin on Titan. Bridge into titan chain from Ethereum to Titan through Titan Power flow bridge https://powerflow.asia/bridge",
+      "denom_units": [
+        {
+          "denom": "factory/titan1pvrwmjuusn9wh34j7y520g8gumuy9xtl3gvprlljfdpwju3x7ucsgehpjy/usdc",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "factory/titan1pvrwmjuusn9wh34j7y520g8gumuy9xtl3gvprlljfdpwju3x7ucsgehpjy/usdc",
+      "name": "USDC",
+      "display": "usdc",
+      "symbol": "USDC",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
```

Comment:
The validation script is picking up on an invalid reference. USDC is registered to Ethereum as  0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 
(Must match letter-casing)